Cargo.toml

```diff
@@ -40,6 +40,7 @@ vec1 = "~1.4.0"
 # ironoxide requires rt-threaded at runtime, but not at compile time
 tokio = {version = "~0.2.0", features=["time", "rt-threaded"]}
 dashmap = "3.4.0"
+ironcore-search-helpers = { git = "https://github.com/IronCoreLabs/search-helpers", rev = "38be96fe6160a739396cf505054a4e0dec6e0aa6", optional=true}
```

Comment:
Is the plan to still add tags to this repo so we can point to a specific tag for versioning?

Reply:
Turns out we have to publish it or we can't publish ironoxide. I'll do that.